tv/src/main/java/io/filmtime/tv/ui/home/HomeScreen.kt

```diff
@@ -70,6 +72,7 @@ fun HomeScreenContent(
         MoviesRow(
           thumbnails = it.items,
           title = it.title,
+          onClick = onNavigateToMovieDetail,
         )
       }
```

Comment:
We are using the name `MoviesRow` for displaying a row of `VideoThumbnail`s which consist of both Movies and Show. 
a) We can rename it to `VidoeThumbnailsRow` or `VidoeThumbnailsSection` or whatever meaningful 

b) There is a bug here that we are using `onNavigateToMovieDetail` for both movie and shows. 
Maybe we can add the type check logic inside MoviesRow or here in home screen:

Approach 1:
```kotlin
MoviesRow(
   onShowClick: (tmdbId) -> Unit,
   onMovieClick: (tmdbId) -> Unit,
)
```

Approach 2:
```kotlin
MoviesRow(
   onItemClick: (type, tmdbId) -> Unit,
)
```
then in HomeScreen:
```kotlin
MoviesRow(
 onItemClick = { type, tmdbId ->
 if type == video => onNavigateToMovieDetail()
if type == show => { // TODO: #ISSUE_NUMBER we will implement show detail screen in another issue}
}
)
```